### DISCUSSION TITLE
[@types/enzyme] ReactWrapper missing renderProp method

### DISCUSSION THREAD

**Discussion by Chris-Thompson-bnsf** [DefinitelyTyped/DefinitelyTyped#59644] [Issues with a @types package]
Related: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53412

The above appears to have been closed in error by the bot - I would guess because there was a PR that referenced the issue. This type is still missing in [@types/enzyme] for enzyme v3.8.0+.

https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/renderProp.html

[a comment by an automated account is not shown]

**Comment by ljharb** [comment]
Sounds like someone should make a PR to add the type.

**Reply by MysteryBlokHed** [reply]
I could open a PR in a couple of hours unless someone else could do one now

**Reply by Chris-Thompson-bnsf** [reply]
I took a stab at this in #59648.
